src/app/components/auth/AuthInitSignUp.tsx

```diff
@@ -107,6 +109,7 @@ const AuthInitSignUp: React.FC = () => {
           required
           placeholder='Password'
           autoComplete='new-password'
+          pattern={PASSWORD_REGEX}
```

Comment:
Instead of regex, we'll be utilizing the `rules` attribute to provide a validation function, that will try to match against all the criteria (minLength, number/upper-lower case inclusion etc. based on the rules set in our Cognito userpool) and inform about the ones the current password is missing. 

For now, let's keep it simple and avoid regex. AWS throws password specific error that we're presenting as a toast, so that should be informative enough for the user until we go into the specifics and handle from the front end (which should be done and will certainly be done as well in the future).